examples/dmd/nonlinear_elasticity.py

```diff
@@ -0,0 +1,629 @@
+'''
+   libROM MFEM Example: nonlinear_elasticity (adapted from ex10p.cpp)
+
+
+      This examples solves a time dependent nonlinear elasticity
+      problem of the form dv/dt = H(x) + S v, dx/dt = v, where H is a
+      hyperelastic model and S is a viscosity operator of Laplacian
+      type.
+
+   How to run:
+      mpirun -np 8 python <arguments>
+
+      Example 1 of arguments:
+      nonlinear_elasticity.py -s 2 -rs 1 -dt 0.01 -tf 5 -visit
+   
+      Output 1:
+      Relative error of DMD velocity (v) at t_final: 5.000000 is 1.145E-03
+      Relative error of DMD position (x) at t_final: 5.000000 is 7.371E-05
+
+      Example 2 of arguments:
+      nonlinear_elasticity.py -s 2 -rs 1 -dt 0.01 -tf 5 -nwinsamp 10 -visit
+   
+      Output 2:
+      Relative error of DMD velocity (v) at t_final: 5.000000 is 1.106E-04
+      Relative error of DMD position (x) at t_final: 5.000000 is 4.304E-06
```

Comment:
Again, I am getting the following in quartz, using 8 processes and pdebug:

Relative error of DMD velocity (v) at t_final: 5.000000 is 6.509E-04
Relative error of DMD position (x) at t_final: 5.000000 is 9.197E-06